e2e/features/column-array-basic.old.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Column Array Basic
+
+	Scenario: Editing and saving do work
+		Given I am on "column-array-basic"
+		When I click cell "Strings"[0]
+		Then Edit form is opened
+		When I enter text
+		Then Value is added
+		When I click delete-icon
+		Then Value is deleted
+		When I press Enter
+		Then Changes are saved
+
+
+
+	Scenario: Custom Template column does work correctly - delete
+		Given I am on "column-array-basic"
+		When I click Custom Template data cell
```

Comment:
why not the same `I click cell "Strings"[0]` ?

Reply:
Old feature file was deleted